.github/workflows/deploy-frontend-pr-previews.yml

```diff
@@ -388,6 +383,8 @@ jobs:
             --port=8000 \
             --cpu=1000m \
             --memory=256Mi \
+            --update-secrets=/etc/env/.env=sta-env-netmanager:latest \
```

Comment:
Hi @Psalmz777 , why am I seeing `netmanager` being mentioned in a PR for` inventory`?